_inc/lib/admin-pages/class.jetpack-my-jetpack-page.php

```diff
@@ -1,187 +0,0 @@
-<?php
```

Comment:
I think problems before came because of cached PHP filed that were `requiring` the removed php files.  We haven't required or included this one for many releases, so it should be ok.